app/src/androidTest/java/org/wikipedia/EspressoLogger.kt

```diff
@@ -0,0 +1,13 @@
+package org.wikipedia
+
+import android.util.Log
+
+class EspressoLogger {
```

Comment:
Can this just be an `object`, then? (instead of a class with companion object)

Reply:
done.